README.md

```diff
@@ -29,50 +42,51 @@ export async function greeting(/** @type {Context} */ context) {
 }
 ```
 
-To run: `./boltzmann.js`. And to view the response: `http GET localhost:5000/hello/world`
+To run: `./boltzmann.js`. And to view the response: `[http](https://httpie.org) GET localhost:5000/hello/world`
```

Comment:
We should tell people to use `curl` here, imo. (i.e., beginners aren't going to know that httpie is an optional step. Let's get people to the response first, then make tooling recommendations.)